Cloudflare/gateway_dns/_meta/manifest.yml

```
@@ -0,0 +1,11 @@
+uuid: 7b1317ec-3f87-4b53-9b6d-3f79045f28fa
+name: Cloudflare Gateway DNS
+slug: cloudflare-gateway-dns
+
+description: >-
+  DNS queries inspected by Cloudflare Gateway
+
+data_sources:
+  Web logs: Log every event and request
+  DNS records: DNS Gateway logs
+  Authentication logs: Access requests are logged
```

Only logs dns questions and answers. I suggest deleting others datasources.
```suggestion
  DNS records: DNS Gateway logs
```